app/views/admin/media/_media_table.html.erb

```diff
@@ -48,7 +48,14 @@
     <% @media.each do |medium| %>
     <tr>
       <td>
-        <a href="<%= url_for(medium.file) %>" rel="noopener" target="_blank"><%= medium.name %></a>
+        <% if medium.file.image? %>
+          <%= image_tag medium.file.variant(resize_to_limit: [100, 100]) %><br>
```

Comment:
Let me know if you want a smaller preview

Reply:
We could introduce a helper for this one, a method in `Medium` model or a presenter. As a matter of fact on edge rails there's a better way handling this by defining the possible variants in the attachment definition within the model.

Reply:
Let's make this 256x256. What do you think ? It's common variant in the admin panel.
If variant definition is available in Rails 6 too, then let's make it a task ;)
Also, how about making the image a link too ? same as the one for the image name.

Reply:
Unfortunately [it isn't,](https://edgeguides.rubyonrails.org/active_storage_overview.html#:~:text=You%20can%20configure%20specific%20variants%20per%20attachment%20by%20calling%20the%20variant%20method%20on%20yielded%20attachable%20object%3A) but we could take a look by monkey-patching it here in the future.

Sure, I will take care of both